Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -270,6 +270,7 @@
     - MechBoards
     - ShuttleBoards
     - CircuitAdventureDynamic # Adventure
+    - LathPlus #CyberneticGenesis
```

Комментарий `Adventure` нужен для того, чтобы случайно не удалить изменение во время апстрима
```suggestion
    - LathPlus # Adventure CyberneticGenesis
```